src/components/time-to-live.tsx

```diff
@@ -4,19 +4,23 @@ import { formatDuration, intervalToDuration } from 'date-fns'
 import { useTimeToLive } from '../hooks/useTimeToLive'
 
 type TimeToLiveProps = {
-	batchId: string
+	batchId: string,
+	amount?: number
 }
 
-export const TimeToLive = ({ batchId }: TimeToLiveProps) => {
-	const ttl = useTimeToLive(batchId)
+export const TimeToLive = ({ batchId, amount = 0 }: TimeToLiveProps) => {
+	const ttl = useTimeToLive(batchId, amount)
```

Comment:
```suggestion
	const ttl = useTimeToLive(batchId)
```
based on the comment in `useTimeToLive`.